src/Entities/Pawn.h

```diff
@@ -33,6 +33,16 @@ class cPawn:
 	virtual void HandleFalling(void);
 	virtual void OnRemoveFromWorld(cWorld & a_World) override;
 
+	/** Handles farmland trampling when hitting the ground.
+	Algorithm:
+	fall height <= 0.6875 blocks: no trampling
+	fall height > 0.6875 and <= 1.0625: 25% chance of trampling
```

Comment:
Your implementation of this gives a different probability - 0.25 * 0.66 as the second else also applies when it gets through the first.

Reply:
Yep much better now.